app/controllers/application_controller.rb

```diff
@@ -13,19 +13,11 @@ class ApplicationController < ActionController::Base
 
   # :nocov:
 
-  def append_to_log(params)
-    @custom_log_items ||= {}
-    @custom_log_items.merge!(params)
-  end
-
   # Looks rather strange, but this is the suggested mechanism to add extra data
   # into the event passed to lograge's custom options. The method is part of
   # Rails' instrumentation code, and is run after each request.
   def append_info_to_payload(payload)
```

Comment:
The behaviour of adding custom fields to lograge logs might be broken with this change.

It is easy given that this is a private method in Rails, and we don't have tests for this behaviour. Sadly is how lograge recommends doing it.

I couldn't see a way to test this when I was looking into it recently, we should probably expand the comment to say how easy is to break it.